send_rx_every_page_before_render_hook.php

```diff
@@ -0,0 +1,52 @@
+<?php
+	return function($project_id){
+        parse_str($_SERVER['QUERY_STRING'], $qs_params);
+     
+	    if($qs_params['page'] == 'patient_screening_info'){
```

Comment:
Let's unhardcode `patient_screening_info` by getting instrument name from `$Proj->metadata['send_rx_prescriber_id']['form_name']`. This way the system still works after renaming the instrument.